### PR TITLE
Rod bromberg js ii

### PR DESCRIPTION
**Pull request by RodBromberg**
@ielvisd

### CODE REVIEW

File: assignments/callbacks.js [hunk 1]
```diff
@@ -23,30 +23,115 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
```
Comment (ielvisd) [inline assignments/callbacks.js:25]: Add console logs because how else are you checking that these work?

File: assignments/array-methods.js [hunk 2]
```diff
@@ -2,82 +2,116 @@
...
-{"id":44,"first_name":"Jodie","last_name":"Stawell","email":"jstawell17@yale.edu","shirt_size":"S","company_name":"Jaxspan","donation":262},
-{"id":45,"first_name":"Falito","last_name":"Karsh","email":"fkarsh18@pcworld.com","shirt_size":"S","company_name":"Mycat","donation":239},
-{"id":46,"first_name":"Reginauld","last_name":"Purselowe","email":"rpurselowe19@thetimes.co.uk","shirt_size":"L","company_name":"Jabbersphere","donation":11},
-{"id":47,"first_name":"Vida","last_name":"Tydd","email":"vtydd1a@dropbox.com","shirt_size":"S","company_name":"Quaxo","donation":55},
-{"id":48,"first_name":"Anderea","last_name":"MacGiolla Pheadair","email":"amacgiollapheadair1b@xing.com","shirt_size":"2XL","company_name":"Kwimbee","donation":214},
-{"id":49,"first_name":"Bel","last_name":"Alway","email":"balway1c@ow.ly","shirt_size":"S","company_name":"Voolia","donation":107},
-{"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
+const runners = [{ "id": 1, "first_name": "Charmain", "last_name": "Seiler", "email": "cseiler0@wired.com", "shirt_size": "2XL", "company_name": "Divanoodle", "donation": 75 },
```
Comment (ielvisd) [inline assignments/array-methods.js:5]: Great work on your array methods.

File: assignments/closure.js [hunk 3]
```diff
@@ -1,11 +1,28 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function foo() {
```
Comment (ielvisd) [inline assignments/closure.js:3]: add console logs to check your code and follow the usage guidelines in the code to check for funcitonality

**Review by ielvisd — APPROVED** [review]
Great work overall, I'll send you the solution code so you can check your work against that. Nice Rod.

File: assignments/callbacks.js (hunk 1 above)
Comment (RodBromberg) [reply assignments/callbacks.js:25]: I use the console almost exclusively then copy and paste into vs code. Been working with quokka more recently but with the call backs just had a few arrays set up and would call the function and pass it in and in the console you don’t need to explicitly type out console.log to check a function

**Pull request merged**